app/react/src/server/framework-preset-cra-styles.js

```diff
@@ -0,0 +1,13 @@
+import { logger } from '@storybook/node-logger';
+import { applyCRAWebpackConfig, isReactScriptsInstalled } from './cra_config';
+
+export function webpackFinal(config) {
```

Comment:
`webpackFinal` kicks in after all the webpack configuration manipulations. I will commit my stashed changes into your PR to fix the webpackFinal phase. It will solve https://github.com/storybooks/storybook/issues/4296#issuecomment-431095849 and the similar issues for CRA that will possibly come.

Reply:
When I used `webpack()` none of the default styles had been loaded in yet, so based off of the angular-cli presets I chose `webpackFinal()`. My understanding was that the `presets` aren't documented yet so I wasn't sure if there were other options.

Is `webpackFinal()` the right choice here now that you've pushed your fix?

Reply:
Yes, that's the place for now, and since it is still internal , we can change this whenever we need =)